community/keyboard_mouse.py

```diff
@@ -3,6 +3,9 @@
 import controller
 import math
 
+W_pressed = False
+LCTRL_pressed = False
```

Comment:
Could these globals be added to the `Keyboard_Mouse` class instead?

Reply:
I tried, it's not working

Reply:
Already made two into one